libs/payments/cart/src/lib/cart.service.ts

```diff
@@ -300,6 +296,18 @@ export class CartService {
 
     // PayPal payment method collection
     if (subscription.collection_method === 'send_invoice') {
+      if (!cart.stripeCustomerId) {
+        throw new CartError('Invalid stripe customer id on cart', {
+          cartId,
+        });
+      }
+      if (subscription.latest_invoice) {
+        const invoice = await this.invoiceManager.retrieve(
+          subscription.latest_invoice
+        );
+        await this.invoiceManager.processPayPalInvoice(invoice);
+      }
```

Comment:
[non-blocking] I figure this will be restructured to be part of checkoutCartWithPaypal in FXA-10603

Reply:
Yes correct. This is adding a tiny bit of scope to FXA-10603 in order to avoid adding a lot of scope to this ticket